src/components/input/Input.tsx

```diff
@@ -9,10 +9,14 @@ type Props = {
   enabled: boolean;
   number: number;
   roundingPrecision?: number;
+  showSigFigs?: boolean;
   setNumber: (newNumber: number) => void;
   setEnabled: (value: boolean) => void;
   showCheckbox?: boolean;
+  showNumberWhenDisabled?: boolean;
   titleTooltip?: string;
+  prefix?: ReactElement;
```

Comment:
"Prefix" doesn't tell me a whole lot, and I need a lot of context to figure it out